### ISSUE TITLE
negative unwarpdir inconsistensies: y- vs. -y

### ISSUE THREAD

**Issue by jokedurnez** [Washington-University/Pipelines#30]
Hi,

I'm running into a problem when running the `PreFreeSurferPipeline.sh` when the unwarp direction is negative on the second axis (y- / -y):

**T2ToT1wDistortionCorrectAndReg.sh** calls FSL's `convertwarp` (and others) requiring the negative direction to be specified as `--unwarpdir=y-`.
But earlier **TopupPreprocessingAll.sh** is called and documentation indicates I should pass  `unwarpdir=-y`.
I've specified `unwarpdir=-y`, and I'm running into an error [on line 270 of T2ToT1wDistortionCorrectAndReg.sh](https://github.com/Washington-University/Pipelines/blob/master/PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh#L270)  I've noticed that most scripts accept both `y-` and `-y`.  Apparently `-y` doesn't pass the FSL commands, but I wonder if `y-` would pass everywhere (also in the other pipelines)...

This is the exact error:
```
Wed Nov 30 05:19:12 UTC 2016 - TopupPreprocessingAll.sh - END: Topup Field Map Generation and Gradient Unwarping

Cannot interpret shift direction = -y
terminate called after throwing an instance of 'NEWMAT::IndexException'
/opt/HCP-Pipelines/PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh: line 252: 24237 Aborted                 ${FSLDIR}/bin/convertwarp --relout --rel --ref=${WD}/Magn
itude --shiftmap=${WD}/FieldMap_ShiftMap${TXw}.nii.gz --shiftdir=${UnwarpDir} --out=${WD}/FieldMap_Warp${TXw}.nii.gz
```

Thanks,

Joke

**Comment by glasserm** [comment]
Are you saying that two of the tools called by the HCP Pipelines are expecting conflicting ways of specifying negation on y or that only one of the directions doesn't work but the other does.

**Comment by jokedurnez** [comment]
`-y` didn't work.  Trying the whole pipeline with `y-` now...

**Comment by glasserm** [comment]
Ok let me know what happens.

**Comment by jokedurnez** [comment]
Oh, I lost track if this, but I can confirm that this worked with 'y-'

**Comment by glasserm** [comment]
I have resolved this with commit:  93865ad
